irohad/main/application.cpp

```diff
@@ -269,11 +258,16 @@ void Irohad::initMstProcessor() {
  * Initializing transaction command service
  */
 void Irohad::initTransactionCommandService() {
-  auto tx_processor =
-      std::make_shared<TransactionProcessorImpl>(pcs, mst_processor);
-
-  command_service = std::make_shared<::torii::CommandService>(
-      tx_processor, storage, std::chrono::seconds(1), 2 * proposal_delay_);
+  status_bus_ = std::make_shared<StatusBusImpl>();
```

Comment:
Maybe move status bus as a separate method? I think it is will be more clear to a reader of the code. And, also, we will introduce status bus as a dependency in other modules.